src/demo/demoUtils.ts

```diff
@@ -426,8 +452,54 @@ export const getToken = (
           },
         })
       }
-      onSuccess(data.message)
+      onSuccess(data.message, data.applicant_id)
     }
   }
   request.send()
 }
+
+export const createCheckIfNeeded = (
+  tokenUrl?: string,
+  applicantId?: string,
+  submittedData?: SdkResponse
+): void => {
+  // Don't create check if createCheck flag isn't present
+  if (!queryParamToValueString.createCheck || !tokenUrl || !applicantId) {
+    return
+  }
+
+  const { poa, docVideo, faceVideo } = queryParamToValueString
+
+  const request = new XMLHttpRequest()
+
+  request.open('POST', tokenUrl.replace('sdk_token', 'check'), true)
```

Comment:
Interesting. So you can trigger a check creating from token factory?

Reply:
Yep, it was there from beginning I believe

Reply:
Ah yes, it was used on mobile SDKs